packages/@coorpacademy-components/src/molecule/discipline-associated-skills/index.js

```diff
@@ -52,7 +52,7 @@ const DisciplineAssociatedSkills = (props, context) => {
         <div className={style.chipsWrapper}>
           {skills.map(skill => {
             function handleSkillClick() {
-              return onSkillClick(skill.ref);
+              return onSkillClick(skill.id);
```

Comment:
Working, still do think a anchor should be set to have the standard html behavior: cursor, and right click (with open in new tab and co behavior)